stdlib/public/TensorFlow/Parameterized.swift

```diff
@@ -65,3 +65,59 @@ public extension Parameterized where Parameters : ParameterGroup {
     allParameters.update(withGradients: gradients, updater)
   }
 }
+
+//===----------------------------------------------------------------------===//
+// Extend common types to conform to ParameterGroup and Parameterized
+//===----------------------------------------------------------------------===//
+
+/// Tensors conform to the protocols by "identity".
+extension Tensor : Parameterized {
+  public typealias Parameters = Tensor<Scalar>
+
+  public var allParameters: Parameters {
+    get {
+      return self
+    }
+    set(newValue) {
```

Comment:
```suggestion
    set {
```
The default new variable name is `newValue`. Only explicit name it when a different name is desired.